data-serving/scripts/convert-data/converters.py

```diff
@@ -77,7 +77,15 @@ def convert_age_range(ages: Any) -> Dict[str, float]:
         }
         where start == end if value represents a single age.
     '''
-    return convert_range(ages, parse_age, lambda x: x)
+    age_range = convert_range(ages, parse_age, lambda x: x)
+    if age_range is not None:
+      has_start = 'start' in age_range.keys()
+      has_end = 'end' in age_range.keys()
+      if ~has_start & has_end:
```

Comment:
prefer boolean operators really:

if has_start and has_end:
...
elif has_start and not has_end"
...

Reply:
Done